webapp/package.json

```diff
@@ -9,32 +9,52 @@
         "fix": "eslint --ignore-pattern node_modules --ignore-pattern dist --ext .js --ext .jsx . --quiet --fix"
     },
     "dependencies": {
-        "core-js": "3.1.4",
+        "@primer/octicons-react": "10.1.0",
+        "core-js": "3.6.5",
+        "csstype": "3.0.3",
         "mattermost-redux": "github:mattermost/mattermost-redux#b56e7e3a6f7f5de281fd3541eefec9345a8bbc89",
         "prop-types": "15.7.2",
-        "react": "16.8.6",
-        "react-bootstrap": "0.32.4",
-        "react-redux": "5.1.1",
-        "redux": "4.0.1"
+        "react": "16.13.1",
+        "react-bootstrap": "1.3.0",
+        "react-custom-scrollbars": "4.2.1",
+        "react-redux": "7.2.1",
+        "redux": "4.0.5"
     },
     "devDependencies": {
-        "@babel/cli": "7.5.0",
-        "@babel/core": "7.5.0",
-        "@babel/plugin-proposal-class-properties": "7.5.0",
-        "@babel/plugin-syntax-dynamic-import": "7.2.0",
-        "@babel/polyfill": "7.4.4",
-        "@babel/preset-env": "7.5.0",
-        "@babel/preset-react": "7.0.0",
-        "@babel/runtime": "7.5.1",
-        "babel-eslint": "10.0.2",
-        "babel-loader": "8.0.6",
-        "css-loader": "3.0.0",
-        "eslint": "6.0.1",
-        "eslint-import-resolver-webpack": "0.11.1",
-        "eslint-plugin-import": "2.18.0",
-        "eslint-plugin-react": "7.14.2",
-        "style-loader": "0.23.1",
-        "webpack": "4.35.3",
-        "webpack-cli": "3.3.5"
+        "@babel/cli": "7.11.6",
+        "@babel/core": "7.11.6",
+        "@babel/plugin-proposal-class-properties": "7.10.4",
+        "@babel/plugin-proposal-nullish-coalescing-operator": "7.17.12",
+        "@babel/plugin-proposal-optional-chaining": "7.11.0",
+        "@babel/plugin-syntax-dynamic-import": "7.8.3",
+        "@babel/polyfill": "7.11.5",
+        "@babel/preset-env": "7.11.5",
+        "@babel/preset-react": "7.10.4",
+        "@babel/preset-typescript": "7.10.4",
+        "@babel/runtime": "7.11.2",
+        "@types/node": "14.11.1",
+        "@types/react": "16.9.49",
+        "@types/react-bootstrap": "0.32.29",
+        "@types/react-custom-scrollbars": "4.0.10",
+        "@types/react-dom": "16.9.8",
+        "@types/react-intl": "3.0.0",
+        "@types/react-redux": "7.1.9",
+        "@types/react-router-dom": "5.1.5",
+        "@types/react-transition-group": "4.4.0",
+        "@typescript-eslint/eslint-plugin": "4.1.1",
+        "@typescript-eslint/parser": "4.1.1",
+        "babel-eslint": "10.1.0",
+        "babel-loader": "8.1.0",
+        "babel-plugin-typescript-to-proptypes": "1.4.1",
+        "css-loader": "4.3.0",
+        "eslint": "7.9.0",
+        "eslint-import-resolver-webpack": "0.12.2",
+        "eslint-plugin-import": "2.22.0",
+        "eslint-plugin-react": "7.20.6",
+        "file-loader": "6.1.0",
+        "style-loader": "1.2.1",
+        "typescript": "4.0.3",
+        "webpack": "4.44.2",
+        "webpack-cli": "3.3.12"
```

Comment:
Why this change?

Reply:
We matched all the dependency versions with github dependencies.